understanding/audio-description-prerecorded.html

```diff
@@ -58,93 +56,73 @@
-                     <li><a href="#test-rules">Test Rules</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
+                     <li><a href="#test-rules">テストルール</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.5:</span>Audio Description (Prerecorded) (Level AA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.5:</span>音声解説 (収録済) (レベル AA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
                   <p>
-                     <a href="#dfn-audio-description">Audio description</a> is provided for all <a href="#dfn-prerecorded">prerecorded</a>
-                     <a href="#dfn-video">video</a> content in <a href="#dfn-synchronized-media">synchronized media</a>.
+                     <a href="#dfn-synchronized-media">同期したメディア</a>に含まれているすべての<a href="#dfn-prerecorded">収録済</a>の<a href="#dfn-video">映像</a>コンテンツに対して、<a href="#dfn-audio-description">音声解説</a>が提供されている。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Videos can be played with audio descriptions.</dd>
+                  <dt>目標</dt>
+                  <dd>音声解説付きで映像を再生できるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Provide a synchronized spoken description of the visual content in videos.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>映像の視覚的な内容に同期した、発話音声による説明を提供する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who cannot see or understand the visual content can hear about it while playing
-                     videos.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>視覚的な内容を見たり理解したりできない人でも、映像を再生しながらその内容を聞くことができる。
```

Comment:
hear it ではなく hear about it で、the visual content の内容を直接聞けるということではなく、内容に関するなにか (解説) を聞くことができるという意味ですね。「内容について」というくらいでしょうか。

```suggestion
                  <dd>視覚的な内容を見たり理解したりできない人でも、映像を再生しながらその内容について聞くことができる。
```

Reply:
@bakera ご指摘ありがとうございます。たしかに、about のニュアンスを訳しきれていませんでした。